### ISSUE TITLE
[Request] Chat Slash Command toggles

### ISSUE THREAD

**Issue by decoytrix** [MidoriKami/Mappy#54]
Using the plugin a few weeks I've noticed features it has the basic game map doesn't and features the basic game map has that the plugin doesn't.  A simpler way to disable and enable the plugin would be beneficial if that could be added. /mappyoff and on, or even a checkbox in the /mappy settings if simpler. It is a nice feature I've seen in a few plugins and it saves a few steps from trying to scrollfind it in the plugins listing when you need to go back to the basic map.

Thanks for consideration.

**Comment by MidoriKami** [comment]
Duplicate: https://github.com/MidoriKami/Mappy/issues/48

**Comment by decoytrix** [comment]
Oh i saw that one. Sorry if I was confusing, but it isn't as issue with hotkeys or integrations. I think I saw that option on by default and know it doesn't fix issues of compatibility with the base game like ones I've listed previously or others like the game's selective quest tracking not working. I was trying to find ways of managing the compatibility tradeoffs since I saw that suggestion didn't work.
